src/components/ui4/ButtonUi4.tsx

```diff
@@ -0,0 +1,154 @@
+import * as React from 'react'
+import { ActivityIndicator, Text, TouchableOpacity, ViewStyle } from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+import { cacheStyles } from 'react-native-patina'
+
+import { usePendingPress } from '../../hooks/usePendingPress'
+import { fixSides, mapSides, sidesToMargin } from '../../util/sides'
+import { Theme, useTheme } from '../services/ThemeContext'
+
+export type ButtonTypeUi4 = 'primary' | 'secondary' | 'tertiary'
```

Comment:
Please don't do "tertiary". We chose the word "escape" because this button style must only be used to leave a scene or modal. We want the styles to have meaning, not just "oooh I randomly like the way this looks." This "escape" makes it clear that this is the button style you use for cancel / exit / etc., and nothing else.

If "tertiary" is something that was decided on logically with the rest of the team, then I would love to hear the logic for when/how these styles should be used.

Reply:
I don't want us to make up new terms (i.e. tile) making it a bit confusing to communicate between devs and everyone else. Primary, secondary, and tertiary are the standard UI/UX terms

https://www.codecademy.com/resources/docs/uiux/button